Makefile

```diff
@@ -25,11 +25,11 @@ APP_LOAD_PARAMS += --appFlags 0x800 --path "44'/60'" --path "45'" --curve secp25
 APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
 
 APPVERSION_M     = 1
-APPVERSION_N     = 0
-APPVERSION_P     = 7
+APPVERSION_N     = 1
+APPVERSION_P     = 1
 APPVERSION       = "$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)"
 
-APPNAME = "Lido"
```

Comment:
Is this intentional ? This has an impact on the SET_EXTERNAL_PLUGIN command of the Ethereum app.

Reply:
I will fix it, it was changed for testing purpose